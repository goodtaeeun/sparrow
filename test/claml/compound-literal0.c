struct a {
  int f1;
  int f2;
};
int b() {
  struct a *p = &(struct a){1, 2};
  return p->f1;
}
