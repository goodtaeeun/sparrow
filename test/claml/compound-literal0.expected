/* Generated by CIL v. 1.7.4 */
/* print_CIL_Input is false */

#line 1 "compound-literal0.c"
struct a {
  int f1;
  int f2;
};
#line 5 "compound-literal0.c"
int b(void) {
  struct a *p;
  struct a tmp;

  {
#line 6
    tmp.f1 = 1;
#line 6
    tmp.f2 = 2;
#line 6
    p = &tmp;
#line 7
    return (p->f1);
  }
}
